authority/provisioner/k8sSA.go

```diff
@@ -137,6 +144,21 @@ func (p *K8sSA) Init(config Config) (err error) {
 		p.kauthn = k8s.AuthenticationV1()
 	*/
 
+	// Initialize the x509 allow/deny policy engine
+	if p.x509Policy, err = policy.NewX509PolicyEngine(p.Options.GetX509Options()); err != nil {
+		return err
+	}
+
+	// Initialize the SSH allow/deny policy engine for user certificates
+	if p.sshUserPolicy, err = policy.NewSSHUserPolicyEngine(p.Options.GetSSHOptions()); err != nil {
+		return err
+	}
```

Comment:
I'm not sure if k8ssa should allow user certificates.

Reply:
According to the table in the docs at https://smallstep.com/docs/step-ca/provisioners it can sign them. No revocation nor renew, though.